### ISSUE TITLE
'auto-py-to-exe' is not recognized as an internal command or external, an executable program or a batch file.

### ISSUE THREAD

**Issue by MeblIkea** [brentvollebregt/auto-py-to-exe#296]
**Quick Checks For You**
- [ O ] I have read/searched in the [help post](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe)
- [ O ] I have [searched other issues](https://github.com/brentvollebregt/auto-py-to-exe/issues?q=is%3Aissue+) looking for an issue similar to mine
- [ O ] I have made sure my applicaiton/script runs before trying to package

**Describe the bug**
Hi, when I type `auto-py-to-exe` in CMD, it says `'auto-py-to-exe' is not recognized as an internal command or external, an executable program or a batch file.`.
I check on [nitratine.net](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#pythonpipauto-py-to-exe-is-not-recognised-as-an-internal-or-external-command-operable-program-or-batch-file), I typed the command it says for having Python path, here is the outputs:
```
>>> import os, sys
>>> print(os.path.dirname(sys.executable))
C:\Users\Meb\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0
>>> print (os.path.dirname(sys.executable) + "\\Scripts")
C:\Users\Meb\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\Scripts   
```
and here is my file explorer in `C:\Users\Meb\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0`:

<img width="598" alt="Capture" src="https://user-images.githubusercontent.com/69876322/183240807-538a8d8e-057c-4455-8c47-738425f382fa.PNG">

There are no `Scripts` folder.

**To Reproduce**
1. Go to your Command Prompt
2. Type `pip install auto-py-to-exe` (or `pip3 install auto-py-to-exe`)
3. Type `auto-py-to-exe`

**Expected behavior**
Is says:
`'auto-py-to-exe' is not recognized as an internal command or external, an executable program or a batch file.`

**Screenshots**
<img width="960" alt="Capture" src="https://user-images.githubusercontent.com/69876322/183240934-42d76e04-e083-43c5-bfee-08eeb573214d.PNG">

**Your Environment:**
- Python version (`python --version`): Python 3.10.2
- auto-py-to-exe version (`pip show auto-py-to-exe`):
- PyInstaller version (`pip show pyinstaller`): 2.19.0
- Eel version (`pip show eel`): 0.12.4

**Comment by brentvollebregt** [comment]
I have a funny feeling this may be due to the setup of your environment / usage of Python from the Windows Store.

Can you create a virtual environment and try to install auto-py-to-exe into that - if this works, we can say this is an environment issue.

**Comment by MeblIkea** [comment]
> I have a funny feeling this may be due to the setup of your environment / usage of Python from the Windows Store.
> 
> Can you create a virtual environment and try to install auto-py-to-exe into that - if this works, we can say this is an environment issue.

Ooh yes, I installed it with Microsoft Store because on the past, I didn't check the "Add Python to environment variables" wheb Setup Python, so when I typed python, it open Microsoft Store, and I installed Python from here. 
I'll try to uninstall the Microsoft Store version, and I'll see if it work. (It will probably destroy all my PyCharm interpreters).


So, after finding how Microsoft Store work, I successfully uninstalled python from Microsoft Store, and added it to PATH.
And, it worked! 

Thanks @brentvolletbregt, really cool!
